sdk/storage/Azure.Storage.Files.DataLake/samples/Sample01a_HelloWorld.cs

```diff
@@ -0,0 +1,428 @@
+
+            // Get a reference to a filesystem named "sample-filesystem-acl" and then create it
+            DataLakeFileSystemClient filesystem = serviceClient.GetFileSystemClient("sample-filesystem-acl");
+            filesystem.Create();
+            try
+            {
+                // Create a DataLake file so we can set the Access Controls on the files
+                DataLakeFileClient fileClient = filesystem.GetFileClient("sample-file");
+                fileClient.Create();
+
+                // Make Access Control List and Set Access Control List
+                fileClient.SetAccessControl("user::rwx,group::r--,mask::rwx,other::---");
+
+                // Get Access Control List
+                PathAccessControl accessControlreturn = fileClient.GetAccessControl();
+
+                //Check accessControl permissions
+                Assert.AreEqual("user::rwx,group::r--,mask::rwx,other::---", accessControlreturn.Acl);
+            }
+            finally
+            {
+                // Clean up after the test when we're finished
+                // Clean up after the test when we're finished
+                filesystem.Delete();
+            }
+        }
+
+        /// <summary>
+        /// Rename a DataLake file and a DatLake directories in a DataLake Filesystem.
+        /// </summary>
+        [Test]
+        public void Rename()
+        {
+            // Make StorageSharedKeyCredential to pass to the serviceClient
+            string storageAccountName = StorageAccountName;
+            string storageAccountKey = StorageAccountKey;
+            Uri serviceUri = StorageAccountBlobUri;
+            StorageSharedKeyCredential sharedKeyCredential = new StorageSharedKeyCredential(storageAccountName, storageAccountKey);
+
+            // Create DataLakeServiceClient using StorageSharedKeyCredentials
+            DataLakeServiceClient serviceClient = new DataLakeServiceClient(serviceUri, sharedKeyCredential);
+
+            // Get a reference to a filesystem named "sample-filesystem-rename" and then create it
+            DataLakeFileSystemClient filesystem = serviceClient.GetFileSystemClient("sample-filesystem-rename");
+            filesystem.Create();
+            try
+            {
+                // Create a DataLake Directory to rename it later
+                DataLakeDirectoryClient directoryClient = filesystem.GetDirectoryClient("sample-directory");
+                directoryClient.Create();
+
+                // Rename the sample directory
+                directoryClient.Rename("sample-directory2");
+
+                // Delete the sample directory using the new path/name
+                filesystem.DeleteDirectory("sample-directory2");
+
+                // Create a DataLake file.
+                DataLakeFileClient fileClient = filesystem.GetFileClient("sample-file");
+                fileClient.Create();
+
+                // Rename the sample file
+                fileClient.Rename("sample-file2");
```

Comment:
```
DataLakeFileClient renamedFileClient =

PathProperties pathProperties = renamedFileClient.GetProperties()
```

We need to show the user we are returning a FileClient associated with the name File.